### PR TITLE
Add a test-all script to help with local testing

### PR DESCRIPTION
**Pull request by timothyfroehlich**
Adding a script to run all tests, to help ensure that your push will pass presubmits.

Added a bit more to the clean-all script too to better clean up the plugin and build-logic projects.

### CODE REVIEW

**Comment by timothyfroehlich** [comment]
@rylin8 @srahman80 Could one of you take a look soon?

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The DesignCompose Tutorial app shows you the capabilities of DesignCompose throu
 
 You'll work with your own copy of the [Tutorial Figma file](https://www.figma.com/community/file/1228110686419863535/Tutorial-for-Automotive-Design-for-Compose). Create your own by clicking **Open in Figma**, which creates a copy of the file in your account. Once open, identify the Figma Document ID from your new file's URL. It's the alphanumeric string between `file/` and the name of the document. For example:
 
-<pre><code>figma.com/file/<b>ABCDEFG123</b>/File-name</code></pre>
+`<pre><code`>`figma.com/file/<b>ABCDEFG123</b>/File-name</code></pre>
```
Comment (rylin8) [inline README.md:35]: Think there is an extra back tick here.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,22 @@ We'll be adding more documentation and guides soon! For now you can look to `ref
...
 # Working with the Source
 
+## Testing presubmits locally
+
+To check that you can pass presubmits and nightlies before pushing to PR you can do the following:
+
+- Prerequisites:
+  - Make sure your system can run Android AVDs
+  - Have the build dependencies below installed
+  - Check out the current supported branch of the AAOS Unbundled repo (see the "Check out the Unbundled AAOS Repo" job in `.github/workflows/main` for the correct branch) and set $ORG_GRADLE_PROJECT_unbundledAAOSDir to the path of the checkout
```
Comment (rylin8) [inline README.md:71]: I'd also mention that you need FIGMA_ACCESS_TOKEN set.

Comment (timothyfroehlich) [reply README.md:71]: Pretty sure that was an accidental change.

File: README.md (hunk 1 above)
Comment (timothyfroehlich) [reply README.md]: Pretty sure that was accidental

**Comment by rylin8** [comment]
Changes look good, but when I ran I failed a bunch of tests. I took out @srahman80's latest change https://github.com/google/automotive-design-compose/pull/122/files and that fixed most of them -- Sherief did you update all the serialized files properly? Do you think you can try running Tim's script here?

**Comment by srahman80** [comment]
> Changes look good, but when I ran I failed a bunch of tests. I took out @srahman80's latest change https://github.com/google/automotive-design-compose/pull/122/files and that fixed most of them -- Sherief did you update all the serialized files properly? Do you think you can try running Tim's script here?

Sorry, I missed the mention about this.  I regenerated the serialized files after rebasing from main, and all presubmit checks passed.  

I am getting an error regarding Configuration cache state when I attempt to run the test-all.sh script, but that might be an issue on my end.

**Comment by timothyfroehlich** [comment]
> 

Let me know if you can't get past that, configuration cache should be working fine.

**Comment by srahman80** [comment]
I'm seeing what I assume are the same errors as @rylin8.  I'll open a PR after this is merged to work on getting those fixed.  Approving this PR.

**Review by srahman80 — APPROVED** [review]

**Pull request merged**